pkg/cluster/internal/create/actions/loadbalancer/haproxy.go

```diff
@@ -96,7 +96,7 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 	if err := loadBalancerNode.Command(
 		"/bin/sh", "-c",
 		fmt.Sprintf(
-			"docker run -d -v /kind/haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg:ro --network host --restart always %s",
+			"docker run -d -v /kind/haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg:ro --network host --restart always --name haproxy %s",
```

Comment:
this container is inside a "node" container :-)

Reply:
@aojea the haproxy container is a container that lives inside a node the already has a name with the prefix of the cluster (e.g. `kind-external-load-balancer` node).

There is no risk of conflict if there are two clusters because the containing node will be different

Reply:
/lgtm cancel

Reply:
yeah, I totally missed it, sorry